ZupIT/ritchie-cli
# Check repository structure

### Description

I implemented a validation when adding repositories, so that the user can only add repositories that have valid formulas
closes #698

### How to verify it
To validate, you can try to add any repository that is not a Ritchie formula repository, using the `rit add repo` command. The same, should return an error to the user with a message saying that the chosen repository does not have formulas.

### Changelog

Validation in the rit add repo command, to add only formula repositories
